journal/modernized/03-chapter-2.adoc

```diff
@@ -177,6 +179,11 @@ But will not the Judge of all the earth do right? From your afflicted cousin,
 [.signed-section-signature]
 Ann Branson
 
+--
+
+[.embedded-content-document.letter]
+--
+
 Ninth Month 4th.--Dear sister Lydia now attending Yearly Meeting.
```

Comment:
You did this perfectly, but just FYI... I think both Jared and I (having gotten a little experience with asciidoc) would now change the beginning of this letter to:

`[.signed-section-context-open]`
Ninth Month 4th.

`[.salutation]`
Dear sister Lydia, now attending Yearly Meeting.

Wouldn't you agree @jaredh159 ??  Sorry if these last two comments are confusing to you. I know they are a bit out of the scope of what you are doing, and have more to do with our method of intake.😬